### PR TITLE
t-shirts test added

### PR DESCRIPTION
**Pull request by cristiancobo**

### CODE REVIEW

**Review by holgiosalos — CHANGES_REQUESTED** [review]
run the `lint:fix` command, also, fix the EOF in all the files that you see this warning:
image

nice to see the usage of `Cypress.env`

File: cypress.json [hunk 1]
```diff
@@ -8,3 +8,5 @@
         "nameProduct": "Faded Short Sleeve T-shirts"
     }
 }
+
+
```
Comment (manuelq12) [inline cypress.json]: You don't need this blank spaces for the EOF, just one is okay

File: cypress/page/address/address.page.ts [hunk 2]
```diff
@@ -9,4 +9,5 @@ class Addresses {
         cy.get(this.checkoutButton).click()
     }
 }
-export {Addresses }
\ No newline at end of file
+export {Addresses }
+
```
Comment (manuelq12) [inline cypress/page/address/address.page.ts]: Please erase this blank space, just be sure the EOF is there

File: cypress/page/authentication/authentication.page.ts [hunk 3]
```diff
@@ -15,4 +15,5 @@ class Authentication {
         cy.get(this.buttonSignIn).click();
     }
 }
-export {Authentication }
\ No newline at end of file
+export {Authentication }
+
```
Comment (manuelq12) [inline cypress/page/authentication/authentication.page.ts]: Please erase this blank space, just be sure the EOF is there

File: cypress/page/t-shirts/t-shirts.page.ts [hunk 4]
```diff
@@ -16,4 +16,5 @@ export class Tshirts{
         .get(this.addToCartButton)
         .click();
     }
-  }
\ No newline at end of file
+  }
+
```
Comment (manuelq12) [inline cypress/page/t-shirts/t-shirts.page.ts]: Blank space and also EOF

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/page/index.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+export { PaymentConfirmation } from '../page/payment-confirmation/payment-confirmation.page';
+export { Shipping } from '../page/shipping/shipping.page';
+export { SummaryPayment } from '../page/summary-payment/summary-payment.page';
+export { ModalWindowCart } from '../page/t-shirts/modal-window-cart.page';
+export { Tshirts } from '../page/t-shirts/t-shirts.page';
+
+
+
+
+
```
Comment (manuelq12) [inline cypress/page/index.ts]: Why so many blank spaces?

File: cypress/page/t-shirts/t-shirts.page.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  
+    public selectTshirt(tshirtName:string) {
+      cy.get(this.tshirts)
+        .find('.product-name')
+        .contains(tshirtName)
+        .first()
+        .get(this.addToCartButton)
+        .click();
+    }
+  }
```
Comment (holgiosalos) [inline cypress/page/t-shirts/t-shirts.page.ts:19]: Fix the indentation for next PR

**Review by holgiosalos — APPROVED** [review]

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**